src/Compilers/CSharp/Portable/Syntax/SyntaxNormalizer.cs

```diff
@@ -192,6 +192,12 @@ private int LineBreaksAfter(SyntaxToken currentToken, SyntaxToken nextToken)
                 return 0;
             }
 
+            if (nextToken.IsKind(SyntaxKind.CloseBraceToken) &&
+                IsAccessorListWithoutAccessorsWithBlockBody(currentToken.Parent?.Parent))
```

Comment:
might read better is `IsAccessorListWithNonBlockBodyAccessors`.

Reply:
> might read better is IsAccessorListWithNonBlockBodyAccessors.

That name would be misleading because it implies that there is an accessor

---
In reply to: [534338023](https://github.com/dotnet/roslyn/pull/49495#discussion_r534338023) [](ancestors = 534338023)

Reply:
@CyrusNajmabadi @AlekseyTs may I leave `IsAccessorListWithoutAccessorsWithBlockBody` or you prefer a different name?

Reply:
> may I leave IsAccessorListWithoutAccessorsWithBlockBody or you prefer a different name?

I prefer the current name.

---
In reply to: [534375265](https://github.com/dotnet/roslyn/pull/49495#discussion_r534375265) [](ancestors = 534375265)